Makefile

```diff
@@ -0,0 +1,50 @@
+PG_CONFIG   ?= $(shell which pg_config)
+DISTNAME     = $(shell grep -m 1 '^name' pg_search/Cargo.toml | sed -e 's/[^"]*"\([^"]*\)",\{0,1\}/\1/')
+DISTVERSION  = $(shell cargo tree --prefix=none --edges=normal | grep 'pgrx v' | sed 's/pgrx v//')
```

Comment:
@thory, that you can use in your extensions as it's more universal as it allows namespaces with packages added through it.

Reply:
Ugh, it should go on the next line to detect the `pgrx` version.